docs/docs/guides/extend-schema.mdx

```diff
@@ -0,0 +1,264 @@
+---
+title: How to extend the schema
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+This guide shows you how to extend existing nodes in your Infrahub schema by adding custom attributes and relationships without modifying the original node definition.
+
+## Prerequisites
+
+Before starting this guide, ensure you have:
+
+- An Infrahub instance running with an existing schema
+- The `infrahubctl` CLI tool installed and configured
+- Basic understanding of [Infrahub schema concepts](../topics/schema)
+- A YAML editor with schema validation support (recommended)
+
+## Step 1: Identify the node to extend
+
+First, determine which existing node you want to extend. You'll need the exact node kind, which follows the format `NamespaceNodeName` (for example: `InfraDevice`, `CoreAccount`).
```

Comment:
Not sure if we can or should recommend altering CoreAccount might want to move it to something like InfraLocation or something